pkg/operator/ceph/object/spec.go

```diff
@@ -124,6 +124,12 @@ func (c *clusterConfig) makeChownInitContainer(rgwConfig *rgwConfig) v1.Containe
 }
 
 func (c *clusterConfig) makeDaemonContainer(rgwConfig *rgwConfig) v1.Container {
+	// Liveness probe port
+	port := intstr.FromInt(int(rgwPortInternalPort))
+	if c.clusterSpec.Network.IsHost() {
+		port = intstr.FromInt(int(c.store.Spec.Gateway.Port))
```

Comment:
per discussion we need to understand if the liveness probe works with https (`SecurePort`)

Reply:
it does! with `URISchemeHTTPS`, will send a fix that too